fantasi.cabal

```diff
@@ -1,20 +1,22 @@
 name:                fantasi
 version:             0.1.0.0
--- synopsis:
--- description:
-homepage:            https://github.com/Tuura/fantasi#readme
-license:             BSD3
+synopsis:
+homepage:            https://github.com/tuura/fantasi
+license:             MIT
 license-file:        LICENSE
-author:              Alessandro de Gennaro and Andrey Mokhov
--- maintainer:
-copyright:           Alessandro de Gennaro and Andrey Mokhov
+author:              Alessandro de Gennaro <a.de-gennaro@ncl.ac.uk>,
+                     Andrey Mokhov <andrey.mokhov@gmail.com>,
+                     github: @allegroCoder, github: @snowleopard
+maintainer:          Alessandro de Gennaro <a.de-gennaro@ncl.ac.uk>,
+                     github: @allegroCoder
+copyright:           Tuura, github.com/tuura
 category:            Web
 build-type:          Simple
 extra-source-files:  README.md
 cabal-version:       >=1.10
 
 executable fantasi
-  hs-source-dirs:   fantasi
+  hs-source-dirs:   src/fantasi
```

Comment:
We can probably keep these modules in `src` too, just for the sake of simplicity.